docs/submitter.rst

```diff
@@ -118,9 +120,10 @@ We follow the steps of the OpenID Connect protocol.
 - The OP responds with an ID Token and usually an Access Token, which are validated with configuration provided by ``OIDC_URL``.
 - The RP can send a request with the Access Token to the UserInfo Endpoint.
 - The UserInfo Endpoint returns Claims about the End-User, use claims ``sub``, ``CSCUserName`` or ``remoteUserIdentifier`` to identify the user and start a session.
+- If using LifeScience AAI as authentication provider, then ``AAI_LIFESCIENCE_ROOT_GROUP`` must be set. This configuration is not used with CSC AAI.
```

Comment:
what if we have the need for multiple ROOT groups ?

Reply:
I'm sure we can change the variable to a CSV in that case. A simpler option would be to not have this config, and take the full strings, but they are quite long. Also it takes admin configuration from LS AAI side to apply a new group to a service, so there won't be an event where we suddenly get a new root group without knowing about it.

Reply:
hmm, i am wondering if we need a mapping between projects e.g. LifeScience root group and workflows developed #578 :thinking:

Reply:
I think I would do the workflows differently. Maybe removing the unavailable workflows from the repo completely, or another environment variable.

Reply:
i don't think it is decide we will use separate instances customized per workflow

Reply:
Then we should filter workflows based on the groups / project. It could be implemented at the workflow level, if it would be ok to make the workflow definitions including which projects to hide the workflow from. That would be a generalized way of doing it.

Reply:
We decided to not split the group names and take them as they are, because if we supported multiple root groups, then it would be possible to hijack groups from other root groups by creating subgroups with the names of other root groups